src/common/FilePreviewCommon/FilePreviewCommon.csproj

```diff
@@ -18,25 +18,25 @@
 
   <ItemGroup>
     <None Update="Assets\Monaco\customLanguages\reg.js">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
why this changes?

Reply:
It made me nuts when it needed extra time to build every time I built FileActionsMenu (which depends on Peek, which depends on this)